index.js

```diff
@@ -1,23 +1,20 @@
-var fs = require('fs');
-var path = require('path');
-var Writer = require('broccoli-writer');
-var Promise = require('rsvp').Promise
-var fixturify = require('fixturify')
+var Plugin = require('broccoli-plugin');
+var fixturify = require('fixturify');
 
-Creator.prototype = Object.create(Writer.prototype);
-Creator.prototype.constructor = Creator;
-function Creator (fixture) {
-  if (!(this instanceof Creator)) return new Creator(fixture);
+Fixturify.prototype = Object.create(Plugin.prototype);
+Fixturify.prototype.constructor = Fixturify;
+function Fixturify(fixture, options) {
+  if (!(this instanceof Fixturify)) return new Fixturify(fixture, options);
+  options = options || {};
+  Plugin.call(this, [], {
+    annotation: options.annotation
```

Comment:
I was planning to pass `persistentOutput: true` here, but it complicates the code. Maybe we'll want to modify broccoli-plugin to have another method `initialBuild` in addition to `build`, so that it's easier to do things only once at the beginning.